Makefile

```diff
@@ -306,15 +307,23 @@ docker-images-jaeger-backend-debug: SUFFIX = -debug
 
 .PHONY: docker-images-jaeger-backend docker-images-jaeger-backend-debug
 docker-images-jaeger-backend docker-images-jaeger-backend-debug: create-baseimg create-debugimg
-	for component in agent collector query ingester ; do \
-		docker build --target $(TARGET) \
-			--tag $(DOCKER_NAMESPACE)/jaeger-$$component$(SUFFIX):${DOCKER_TAG} \
+	for component in "jaeger-agent" "jaeger-collector" "jaeger-query" "jaeger-ingester" "all-in-one" ; do \
+		regex="(jaeger-)(.*)"; \
+		component_prefix=""; \
```

Comment:
this var is not needed

Reply:
Thanks, that's simpler: [ae6b0e1](https://github.com/jaegertracing/jaeger/pull/3813/commits/ae6b0e1f79744c70838326b023e897ebe7df8cc1)